### ISSUE TITLE
The docs for atomic lines should be updated

### ISSUE THREAD

**Issue by HajimeKawahara** [HajimeKawahara/exojax#401]
related to #400, we need to update the docs related to the atomic lines.

**Comment by HajimeKawahara** [comment]
any progress? @YuiKasagi

**Comment by YuiKasagi** [comment]
Sorry, I forgot about this. I will check the document and update it within a few days.

**Comment by YuiKasagi** [comment]
I attempted to update the relevant documents (`documents/tutorials/Forward_modeling_for_Fe_I_lines_of_Kurucz.ipynb` and `documents/tutorials/Forward_modeling_for_metal_line.ipynb`) using `OpaDirect`.

However, at the beginning of the tutorial, I noticed a note stating, "Currently, opa cannot be used for metal lines yet." In practice, it indeed did not work. This issue may be related to #539. Has there been any progress on this?

If not, I think there is nothing to update in these documents for now.

**Comment by YuiKasagi** [comment]
Yes, both notebooks work in my environment in release_2_0 branch.
